### PR TITLE
Add fuzzer repro path to nightly runs.

### PR DESCRIPTION
**Pull request by kgpai**
Add fuzzer repro path to the nightly runs.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -122,13 +122,15 @@ commands:
       - run:
           name: "Build Benchmarks - << parameters.benchmark_class >>"
           command: |
-            make benchmarks-basic-build NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
+            make benchmarks-basic-build NUM_THREADS=8 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
```
Comment (mbasmanova) [inline .circleci/config.yml:125]: Unrelated change?

File: .circleci/config.yml [hunk 2]
```diff
@@ -244,7 +248,7 @@ jobs:
       - run:
           name: "Build"
           command: |
-            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8 EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
+            make debug NUM_THREADS=8 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8 EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
```
Comment (mbasmanova) [inline .circleci/config.yml:251]: It would be nice to extract this change into a separate PR.

**Review by mbasmanova — APPROVED** [review]
Looks good, but it seems like there are multiple changes mixed in this PR.

File: .circleci/config.yml (hunk 1 above)
Comment (kgpai) [reply .circleci/config.yml:125]: No its related, reducing the instance size to xlarge which has 8 cores.

**Comment by kgpai** [comment]
Macos failure is due to homebrew having some problems..

**Review by raulcd — APPROVED** [review]
LGTM
